zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/schema/elasticsearch/ElasticsearchEngineClient.java

```diff
@@ -84,6 +84,18 @@ public void createIndexTemplate(
       client.indices().putIndexTemplate(request);
       LOG.debug("Template [{}] was successfully created", templateDescriptor.getTemplateName());
     } catch (final IOException | ElasticsearchException e) {
+      if (e.getMessage()
+          .contains(
+              String.format(
+                  "index template [%s] already exists", templateDescriptor.getTemplateName()))) {
```

Comment:
💭 `ElasticsearchException` rather has an `error()` field that has a `type`. Is there one type that matches our failure here? would be better than matching the error message text

Reply:
💭 Also similar to what we did for `opensearch`, elastic search client has a `existsTemplate()` endpoint. Would using that be cleaner?

Reply:
there is not feature parity regarding the clients that was why opensearch needed to manually implement the create logic. But elastic has a create field so no need.